DebtNote/Controllers/UserController.cs

```diff
@@ -26,39 +26,40 @@ public UserController(ApplicationContext context, IUserService userService)
         [HttpGet("id")]
         [ProducesResponseType(typeof(User), StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        public JsonResult GetById(int id)
+        public IActionResult GetById(int id)
         {
-            return new JsonResult(UserService.GetById(id));//do smth for 404
+            var result = UserService.GetById(id);
+            return result == null ? NotFound() : Ok(result);
         }
 
         [HttpPost]
         [ProducesResponseType(StatusCodes.Status201Created)]
-        public JsonResult CreateUser(User user) => new JsonResult(UserService.CreateUser(user));
+        public IActionResult CreateUser(User user)
+        {
+            UserService.CreateUser(user);
+            return CreatedAtAction(nameof(GetById),new {id=user.Id},user);
```

Comment:
add space after commas, add spaces before and afrer `=`, don't forget to press Ctrl K Ctrl D every time!